src/metrics/log_recorder.rs

```diff
@@ -0,0 +1,41 @@
+use crate::application::conf::logger::ArconLogger;
+use metrics::{GaugeValue, Key, Recorder, Unit};
+
+pub struct LogRecorder {
+    pub(crate) logger: ArconLogger,
+}
+
+impl Recorder for LogRecorder {
+    fn register_counter(&self, key: &Key, unit: Option<Unit>, description: Option<&'static str>) {
+        println!(
```

Comment:
the logger is not being used 😄  Just replace the println!  with ``info!(self.logger,  ....);``